fenix/app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -855,12 +856,27 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
         }
     }
 
-    private fun deviceRamBytes(): Long {
+    @VisibleForTesting
+    internal fun getDeviceAvailableMemoryInGB(): Long {
```

Comment:
```suggestion
    internal fun getDeviceTotalRAM(): Long {
```

or something like that